scripts/js/lib/api/generateToc.ts

```diff
@@ -64,14 +66,15 @@ export function generateToc(pkg: Pkg, results: HtmlToMdResultWithUrl[]): Toc {
 
 function getModulesAndItems(
   results: HtmlToMdResultWithUrl[],
+  options: { isCApi: boolean },
 ): [HtmlToMdResultWithUrl[], HtmlToMdResultWithUrl[]] {
   const resultsWithName = results.filter(
     (result) => !isEmpty(result.meta.apiName),
   );
 
-  const modules = resultsWithName.filter(
-    (result) => result.meta.apiType === "module",
-  );
+  const modules = options.isCApi
+    ? resultsWithName
+    : resultsWithName.filter((result) => result.meta.apiType === "module");
```

Comment:
A comment may help, like

// In Python, we want to group by modules for better organization.
// Meanwhile, C does not have the idea of modules.